### ISSUE TITLE
when receiving notifications in the system tray NVDA does not read them it just plays the ding sound

### ISSUE THREAD

**Issue by armstrong25** [Orinks/AccessiWeather#361]
when i get updates such as when the forecast discussion gets updated and I've got the app running in the system tray NVDA does not read me what's updated, the ding sound just plays and thats it

---
**System Information:**
```
- App Version: 0.4.3
- OS: Windows 10
- Python: 3.12.10
```
